### DISCUSSION TITLE
Running vite 5 and vue 3.3 breaks

### DISCUSSION THREAD

**Discussion by tromidas15** [vitejs/vite-plugin-vue#331] [Q&A]
If I try to run (an options api project) on vue 3.3 I get  [vite] Pre-transform error: Failed to parse source for import analysis because the content contains invalid JS syntax. Install @vitejs/plugin-vue to handle .vue files. On vue 3.2 is fine. Any idea why ?

**Comment by grctest** [comment]
Hey, I'm also experiencing this error: https://github.com/vitejs/vite-plugin-vue/discussions/329

**Comment by jackonwei** [comment]
Vue 3.2.47 also reports the same error.
